run-make-check.sh

```diff
@@ -123,7 +123,7 @@ EOM
         echo "Current ccache max_size setting:"
         ccache -p | grep max_size
     fi
-    $DRY_RUN ccache -z # Reset the ccache statistics
+    $DRY_RUN ccache -sz # Reset the ccache statistics & show the current configuration
```

Comment:
This is just a nit, but I don't like `&` ampersands in code comments (the ampersand seems like a special character, so my eyes trip over it when reading). Would you mind replacing the ampersand with the word "and"?